css/mathfield.less

```diff
@@ -60,7 +60,7 @@
   flex-flow: row;
   justify-content: space-between;
   align-items: flex-end;
-  min-height: 39px; /* Need some room for the virtual keyboard toggle */
+  min-height: 12px; /* Need some room for the virtual keyboard toggle */
```

Comment:
Doesn't this cause a problem, as per the comment on that line? (that is the height must be large enough to have room for the virtual keyboard toggle)

Reply:
That's true, I haven't properly tested it with the virtual keyboard toggle. My app doesn't have one after all.

I changed the approach to always make the `ML__fieldcontainer__field` as large as the `ML__fieldcontainer`. Since the `ML__fieldcontainer` contains both the `ML__fieldcontainer__field` and the virtual keyboard toggle, this works out fine.